### ISSUE TITLE
Unchecked 'Notify Quantity Use Default' for product when custom source added

### ISSUE THREAD

**Issue by smoskaluk** [magento/inventory#1647]
### Preconditions

1. Magento MSI single stock mode
2. Simple product created with some positive qty

### Steps to reproduce

1. Create additonal source
2. Open your simple product page in admin and go ot 'Sources' tab

### Expected result

1. Qty allocated on default source and  'Notify Quantity Use Default' for Default source is checked 

### Actual result

1. 'Notify Quantity Use Default' for Default source is unchecked 

![image](https://user-images.githubusercontent.com/27818516/45618533-b0397000-ba7e-11e8-96a1-ec70b23fd259.png)

**Comment by VitaliyBoyko** [comment]
Hi 
This is might be solved in #1453 which is not merged yet.
Better to postpone this till mentioned PR not being merged to avoid double work.

**Comment by VitaliyBoyko** [comment]
@smoskaluk Already on me :)

**Comment by smoskaluk** [comment]
@VitaliyBoyko I've checked - #1453 doesn't fix this problem.

**Comment by VitaliyBoyko** [comment]
@smoskaluk Thank you. I gonna take care of this.

**Comment by maghamed** [comment]
@VitaliyBoyko just merged #1453, so you are unblocked to take care of this issue
